components/table/TableHead.js

```diff
@@ -5,7 +5,8 @@ const factory = (Checkbox) => {
     let selectCell;
     const contentCells = Object.keys(model).map((key) => {
       const name = model[key].title || key;
-      return <th key={key}>{name}</th>;
+      const width = model[key].width;
+      return <th key={key} style={{width}}>{name}</th>;
```

Comment:
I think is way more handy if instead of using a `width` you use a `style` tag. Anyway, you are applying a style from the model to the Head. Not sure if it should be applied in the body too. Or, in case it doesn't, it should be `headStyle` and `style`.

In any case, this implementation is probably going to change quite soon. It is not really flexible if you want to do very specific customizations.